### PR TITLE
Changed griffincamp to la commune in footer

### PR DESCRIPTION
**Pull request by nk1tz**
Hi Ziad,

I found two instances of GriffinCamp and changed them both to "La Commune".
I also changed the `<a href=...` to www.lacommune.ca

### CODE REVIEW

**Comment by ziad-saab** [comment]
This is a great start. I forgot to mention that there is another "griffin camp" somewhere in the codebase. Can you find it and modify it as well please?

**Comment by nk1tz** [comment]
Hi Zihad, I have addressed your concern. see below.

File: src/about/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h1 class="intro-section__headline">About Us</h1>
   <div class="section-content">
     <h1 class="about-page__heading">About DecodeMTL</h1>
     <p class="about-page__text">
-      We are currently running our courses out of the GriffinCamp a leading startup accelerator in downtown Montreal's innovation district.
+      We are currently running our courses out of "La Commune" a leading startup accelerator in downtown Montreal's innovation district.
       Our goal is to bring coding literacy to as many people as possible. We believe that with the right environment, best teachers,
```
Comment (ziad-saab) [inline src/about/index.html:17]: Can you find a better description of La Commune please? Changing the name but keeping the old description seems wrong.

**Comment by nyanofthemoon** [comment]
Looking good! There are actually two other instances hidden in the codebase though. Can you find them all? (Hint: see other pull requests) ;)

**Comment by nk1tz** [comment]
Hey I found them!

File: src/_data/faq.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     {
       "question": "If I have questions after class time, is someone able to help me?",
       "answer": "Yes, our instructors have office hours where they will happily assist you with any question or technical problem you may have encountered."
-    },
+    }
```
Comment (ziad-saab) [inline src/_data/faq.json]: You seem to have removed a `,` here. Can you please check??

**Comment by ziad-saab** [comment]
This is great! I would merge this if it was real. :+1:

**Pull request closed without merging**